packages/SidePanel/src/hooks/useFooterOffset.js

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import debounce from "lodash.debounce";
 
 const useFooterOffset = (height, ref = null, isSticky) => {
```

Comment:
can you check if `useFooterOffset` this is used? If not we can just delete the whole thing